kubejs/server_scripts/base/tags/items/forge/stained_glass_pane.js

```diff
@@ -0,0 +1,5 @@
+ ServerEvents.tags('item', event => {
+    for (var i = 0; i < colors.length; i++) {
```

Comment:
forEach here? :)

Reply:
This whole thing can actually go. They now have `forge:glass_panes/color`, `forge:glass_panes`, and `forge:stained_glass_panes` on them by default.